contentcuration/contentcuration/templatetags/perseus_tags.py

```diff
@@ -1,15 +1,10 @@
-import re
+import json
 from django import template
 from django.template.defaultfilters import stringfilter
 
 register = template.Library()
 
 @register.filter
 @stringfilter
-def escape_newline(value):
-	return value.replace('\n', '\\n')
-
-@register.filter
-@stringfilter
 def escape_chars(value):
-	return value.replace('\n', '\\n').replace('\\"', '"')
\ No newline at end of file
+    return json.dumps(value)[1:-1]
```

Comment:
Right now, the [[☃ numeric-input 1]] or [[☃ radio 1]] is added to the end, so it needs to be added into the string. Alternatively, I could add it during the escape_chars method, but it seems to add some complexity to the method